scripts/benchmark_stats.py

```diff
@@ -69,6 +76,12 @@ def _process_file_data(self):
                         )
                         self._total_failures += int(failure_count)
                     else:
+                        if row["Name"] == "/submitted/download-pdf":
+                            self._pdf_percentile.append(int(row.get("99%")))
```

Comment:
Same as [the other comment](https://github.com/ONSdigital/eq-survey-runner-benchmark/pull/164/files#r1714766385). Wouldn't it be better to use `round(x)` since it's more accurate than just cropping? Also Line 77 should be fixed as well?

Reply:
Not sure for line 77 as it should be a whole number? Updated line 80 however 👍